### ISSUE TITLE
sbcli list has a long pause when no config present

### ISSUE THREAD

**Issue by jmrodri** [automationbroker/sbcli#4]
When there is no configuration for sbcli, calling `sbcli list` will print out:
```
Didn't find config file, creating one.
```
Then there is a long pause while it works. Before dumping out the apbs. Feels like we should have a progress bar or something to indicate that sbcli is creating a new config.

```
Didn't find config file, creating one.
[####           ]
```

**Comment by dymurray** [comment]
```
Didn't find config file, creating one.
Bootstrapping bundle specs from registry <registry info>
[###       ]
```

**Comment by djwhatle** [comment]
PR 13 included a progress indicator due to the "info" loglevel being set. The user can see logs from bundle-lib.
https://github.com/automationbroker/sbcli/pull/13/files#diff-ff7686b39bf90dc2520886fb874371a4R59

It's not the prettiest but it is pretty much constant output:
```
[dwhatley@precision-t sbcli]$ ./sbcli registry add --name aws --org awsservicebroker --type dockerhub
[dwhatley@precision-t sbcli]$ ./sbcli bundle list
Getting specs for registry: [aws]
INFO == REGISTRY CX ==                            
INFO Name: aws                                    
INFO Type: dockerhub                              
INFO Url: docker.io                               
INFO APBs filtered by white/blacklist filter:-> awsservicebroker/aws-service-broker 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/athena-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/lex-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/sqs-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/dynamodb-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/sns-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/emr-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/kms-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/kinesis-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/redshift-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/rdspostgresql-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/elasticache-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/rekognition-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/rdsmysql-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/s3-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/rdsmariadb-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/polly-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/translate-apb:latest runtime is 1 
INFO No runtime label found. Set runtime=1. Will use 'exec' to gather bind credentials 
INFO adapter::imageToSpec -> Image docker.io/awsservicebroker/route53-apb:latest runtime is 1 
INFO Validating specs...                          
INFO All specs passed validation!                 
INFO Registry aws has 18 bundles available from 19 images scanned 
 BUNDLE             IMAGE                                                    REGISTRY  
 -------------  -+- ---------------------------------------------------  -+- ---  
 sqs             |  docker.io/awsservicebroker/sqs-apb:latest             |  aws  
 lex             |  docker.io/awsservicebroker/lex-apb:latest             |  aws  
 dynamodb        |  docker.io/awsservicebroker/dynamodb-apb:latest        |  aws  
 athena-apb      |  docker.io/awsservicebroker/athena-apb:latest          |  aws  
 rekognition     |  docker.io/awsservicebroker/rekognition-apb:latest     |  aws  
 elasticache     |  docker.io/awsservicebroker/elasticache-apb:latest     |  aws  
 polly           |  docker.io/awsservicebroker/polly-apb:latest           |  aws  
 kinesis         |  docker.io/awsservicebroker/kinesis-apb:latest         |  aws  
 rdsmariadb      |  docker.io/awsservicebroker/rdsmariadb-apb:latest      |  aws  
 kms             |  docker.io/awsservicebroker/kms-apb:latest             |  aws  
 translate       |  docker.io/awsservicebroker/translate-apb:latest       |  aws  
 redshift        |  docker.io/awsservicebroker/redshift-apb:latest        |  aws  
 s3              |  docker.io/awsservicebroker/s3-apb:latest              |  aws  
 sns             |  docker.io/awsservicebroker/sns-apb:latest             |  aws  
 emr             |  docker.io/awsservicebroker/emr-apb:latest             |  aws  
 rdspostgresql   |  docker.io/awsservicebroker/rdspostgresql-apb:latest   |  aws  
 route53         |  docker.io/awsservicebroker/route53-apb:latest         |  aws  
 rdsmysql        |  docker.io/awsservicebroker/rdsmysql-apb:latest        |  aws  
[dwhatley@precision-t sbcli]$
```

**Comment by jmrodri** [comment]
Fixed by PR #13
